app/views/allocation_staff/_pom_table_row.html.erb

```diff
@@ -1,4 +1,4 @@
-<tr class="govuk-table__row">
+<tr id=pom-list class="govuk-table__row">
```

Comment:
### ⚠️ Invalid HTML

This view partial represents a single table row, which is used multiple times on the same page – i.e. it's rendered in a loop, once for each POM that needs to be displayed in the table. We should avoid giving elements in this partial a static/fixed ID attribute – because otherwise we'll end up with multiple HTML elements on the page that have the same ID.

For example, the HTML structure will look something like this:

```html
<table>
  <tr id="pom-list" class="govuk-table__row">
    <!-- First POM's table cells -->
  </tr>
  <tr id="pom-list" class="govuk-table__row">
    <!-- Second POM's table cells -->
  </tr>
  <tr id="pom-list" class="govuk-table__row">
    <!-- Third POM's table cells -->
  </tr>
</table>
```

HTML IDs must be unique on the page – so you shouldn't have multiple elements with the same `id` attribute. It's fine for multiple elements to use the same `class` though – so that might be a better thing to use here.

Also, HTML attribute values should have `"` double quotes around them – e.g. `id="pom-list"` rather than `id=pom-list`.

Browsers are very forgiving when it comes to parsing invalid HTML – they usually just figure out what we meant and don't complain about it. But nonetheless, we shouldn't knowingly write invalid HTML.

Reply:
Thanks @ollietreend, yes that makes sense.